<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>987</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vector Trace</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushClear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAnimate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Animate</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="dispWidget" name="displayW" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>970</width>
      <height>781</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CoordLabel" name="labelCoords">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>631</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="CoordLabel" name="extraInfo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>631</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Default Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushUndo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVector_Trace">
    <property name="title">
     <string>Vector Trace</string>
    </property>
   </widget>
   <addaction name="menuVector_Trace"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>dispWidget</class>
   <extends>QWidget</extends>
   <header>dispwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CoordLabel</class>
   <extends>QLabel</extends>
   <header>coordlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
